src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const handleClick = (tab) => {
+    if (selectedTabId !== tab.id) {
+      onTabSelected(tab);
```

Comment:
It can be simplified by passing the id as an argument, not the whole tab

Reply:
One of the tests failed after I changed this code logic, I suppose it expects me to pass the whole tab object as a parameter to the onTabSelected func, not only an id
<img width="463" alt="image" src="https://github.com/mate-academy/react_tabs-js/assets/88306382/abaad36f-f5bb-446e-b6e8-ad611cd76c11">